tests/integration/targets/nd_compliance_requirement_config_import/tasks/main.yml

```diff
@@ -0,0 +1,192 @@
+# Test code for the ND modules
+# Copyright: (c) 2022, Akini Ross (@akinross) <akinross@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
Add check for insights_group and site_name

Reply:
ok